UI/Pages/SearchResults.cshtml.cs

```diff
@@ -35,6 +35,11 @@ public async Task OnGet(Query Data)
                 ModelState.AddModelError($"Data.{error.PropertyName}", error.ErrorMessage);
     }
 
+    public async Task OnGetClearAllFilters(string postcode)
+    {
+        this.Data = await mediator.Send(new Query { Postcode = postcode, Subjects = null, TuitionType = TuitionType.Both, KeyStages = null });
```

Comment:
There's no unit test covering this code. Please add a simple test or test covering resetting subjects, tuition type and key stages while maintaining postcode

Reply:
Unit test added